tests/e2e/scale/noobaa/test_scale_obc_creation.py

```diff
@@ -0,0 +1,184 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs import constants, scale_noobaa_lib
+from ocs_ci.framework.testlib import scale, E2ETest
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+from ocs_ci.framework.pytest_customization.marks import (
+    vsphere_platform_required,
+    skipif_aws_creds_are_missing,
+)
+
+log = logging.getLogger(__name__)
+
+
+@scale
+class TestScaleOCBCreation(E2ETest):
+    """
+    OBC scale creation, creating up to max support number of OBCs.
+    OBCs are created in the Multicloud Object Gateway and
+    Ceph Object Gateway (RGW)
+
+    """
+
+    namespace = constants.OPENSHIFT_STORAGE_NAMESPACE
+    sc_name = constants.NOOBAA_SC
+    sc_rgw_name = constants.DEFAULT_STORAGECLASS_RGW
+    scale_obc_count = 1000
+    num_obc_batch = 50
```

Comment:
@tiffanyn108 any reason for lesser batch count ?, in PVC creation using kubejob I tried using 750 PVC per batch and it worked without any issues.

Reply:
obc creation in larger batch is having issue which causes obc in unbound state. According noobaa developer, we are supporting up to 1000 obcs but not creating them in a same time or in few seconds apart.